src/status_im/contexts/wallet/create_account/new_keypair/check_your_backup/view.cljs

```diff
@@ -0,0 +1,127 @@
+   [button
+    (assoc params
+           :word         (first options)
+           :margin-right 12)]
+   [button (assoc params :word (second options))]])
+
+(defn- view-internal
+  []
+  (let [random-indices  (random-selection)
+        quiz-index      (reagent/atom 0)
+        incorrect-count (reagent/atom 0)
+        show-error?     (reagent/atom false)]
+    (fn []
+      (let [current-word-index (get random-indices (min @quiz-index 3))
+            current-word       (get temp/secret-phrase current-word-index)
+            options            (random-words-with-string temp/random-words current-word)
+            button-params      {:quiz-index      quiz-index
+                                :current-word    current-word
+                                :incorrect-count incorrect-count
+                                :show-error?     show-error?}]
+        [rn/view {:style {:flex 1}}
+         [quo/page-nav
+          {:icon-name           :i/arrow-left
+           :on-press            #(rf/dispatch [:navigate-back])
+           :accessibility-label :top-bar}]
+         [quo/text-combinations
+          {:container-style style/header-container
+           :title           (i18n/label :t/check-your-backup)
+           :description     (i18n/label :t/confirm-the-position)}]
+         [rn/flat-list
+          {:data                    random-indices
+           :render-fn               (fn [num index]
+                                      [quo/quiz-item
+                                       {:state    (cond
+                                                    (and (= @quiz-index index)
+                                                         (pos? @incorrect-count)
+                                                         @show-error?)
+                                                    :error
+
+                                                    (= @quiz-index index)
+                                                    :empty
+
+                                                    (> @quiz-index index)
+                                                    :success
+
+                                                    :else
+                                                    :disabled)
+                                        :word     (get temp/secret-phrase num)
+                                        :number   (inc num)
+                                        :on-press #(when (= @quiz-index index)
+                                                     (reset! show-error? false))}])
+           :separator               [rn/view {:style {:height 8}}]
+           :content-container-style {:padding-horizontal 20}}]
+         [rn/view {:style style/buttons-container}
+          [buttons-row
+           (assoc button-params
+                  :margin-bottom 12
+                  :options       (subvec options 0 2))]
+          [buttons-row (assoc button-params :options (subvec options 2 4))]]]))))
```

Comment:
We could move the `(subvec options 0 2)` and `(subvec options 2 4)` into let vars for maintainability.

Reply:
@smohamedjavid hmm why should they be moved into let vars? I think it's clear enough

Btw these are the 2 rows of answer options and the bottom of the screen

Reply:
> why should they be moved into let vars? I think it's clear enough. There isn't something being recalculated twice.

Yes, Omar, It's not recalculated. The idea is to have it in let var (stays top) so that in future, If there are any changes, we can change them, instead of going through the UI code to find them.